rvc-implementation.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Description: The true virtcontainers implementation of the RVC interface.
+// This indirection is required to allow an alternative implemenation to be
+// used for testing purposes.
+
+package main
+
+import (
+	"syscall"
+
+	"github.com/Sirupsen/logrus"
+	vc "github.com/containers/virtcontainers"
+)
+
+// virtcontainers implementation type
+type vcImpl struct {
+}
+
+func (impl *vcImpl) SetLogger(logger logrus.FieldLogger) {
+	vc.SetLogger(logger)
```

Comment:
fmi, are we expecting any performance impact at all with the indirection, or do we think either Go will optimise it away, or it will be so small and we make so few actual calls and only at startup that it will be negligible?